datahub-classify/tests/test_similarity_predictor.py

```diff
@@ -0,0 +1,487 @@
+import glob
+import itertools
+import json
+import logging
+import os
+import pickle
+import time
+from typing import Dict, Tuple
+
+import numpy as np
+import pandas as pd
+import pytest
+from sklearn.metrics import classification_report, confusion_matrix
+
+from datahub_classify.helper_classes import (
+    ColumnInfo,
+    ColumnMetadata,
+    TableInfo,
+    TableMetadata,
+)
+from datahub_classify.similarity_predictor import check_similarity
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+logger.info("libraries Imported..................")
+
+SEED = 100
+np.random.seed(SEED)
```

Comment:
why do we need this? where are we using randomness?

Reply:
Randomness is used while creating TableInfo objects from csv files in unit testing. We have defined some sample platform names and randomly select one for each table.

Reply:
please be more specific about where the randomness is happening? we should not be relying on any random behavior in this library or in our tests

Reply:
While creating TableInfo objects for unit testing, the platform information in the TableInfo objects is filled by randomly selecting a "platform_name" from a list of custom platform names. With a fixed seed we are ensuring the reproducibility. 
No we are thinking to use the serialized list of TableInfo objects so we don't need the TableInfo object creation part in the unit test. Randomness is also going to be removed as a result.

Reply:
Serialized the TableInfo objects for test and removed randomness from the code.